google/cloud/storage/transfer_manager.py

```diff
@@ -732,6 +751,8 @@ def download_chunks_concurrently(
     deadline=None,
     worker_type=PROCESS,
     max_workers=DEFAULT_MAX_WORKERS,
+    *,
```

Comment:
This is special Python 3 syntax that marks everything after that point as "keyword argument only". The reason I do that for extra arguments is so that I don't have to worry about ordering or reordering them in future revisions. It doesn't need a name.